src/antidote_pb_process.erl

```diff
@@ -37,24 +37,27 @@
 
 -export([process/1]).
 
--spec decode_clock(binary()) -> snapshot_time() | ignore.
+-spec decode_clock(binary()) -> snapshot_time() | ignore | txid().
```

Comment:
The function name `decode_clock` is a bit misleading if it also handles  `txid`?

Reply:
Renamed to from_bin